kosmorro: add page (#4498)
tldr-pages/tldr: pages/common/kosmorro.md

# kosmorro

> Compute the ephemerides and the events for a given date, at a given position on Earth.

> > More information: http://kosmorro.space.

Reviewer: We forgot to add the `<>` to the URL. Also, the URL redirects to the official repo. Maybe we can use the github URL directly.
```suggestion
> More information: <https://github.com/Deuchnord/kosmorro>.
```

Author: > Maybe we can use the github URL directly.

I'm not quite sure about that, this redirection is temporary, I'm making a website to replace it in the future. Or maybe I should make a PR when it'll be done?

- Get ephemerides for Paris, France:

`kosmorro --latitude={{48.7996}} --longitude={{2.3511}}`

- Get ephemerides for Paris, France, in the UTC+2 timezone:

`kosmorro --latitude={{48.7996}} --longitude={{2.3511}} --timezone={{2}}`

- Get ephemerides for Paris, France, on June 9th, 2020:

`kosmorro --latitude={{48.7996}} --longitude={{2.3511}} --date={{2020-06-09}}`

> - Generate a PDF (note: you need to install TeXLive):

Reviewer 2: Perhaps this will read better?
```suggestion
- Generate a PDF (note: TeXLive must be installed):
```

Generally we try to avoid the 2nd person (e..g `you`, `your`, etc) in tldr pages.
